apps/duck-web/public/pcm16k-worklet.js

```diff
@@ -0,0 +1 @@
+class PCM16kProcessor extends AudioWorkletProcessor {\n  constructor() {\n    super();\n    this._ratio = sampleRate / 16000;\n    this._carry = 0;\n  }\n  process(inputs) {\n    const input = inputs[0]?.[0];\n    if (!input) return true;\n    const ratio = this._ratio;\n    const outLen = Math.floor((this._carry + input.length) / ratio);\n    const out = new Float32Array(outLen);\n    let i = -this._carry;\n    for (let n = 0; n < outLen; n++) {\n      const start = Math.floor((n * ratio + i));\n      const end = Math.floor(((n + 1) * ratio + i));\n      let sum = 0;\n      for (let k = start; k < end; k++) sum = (input[k] || 0) + sum;\n      out[n] = sum / Math.max(1, end - start);\n    }\n    const lastUsed = Math.floor(outLen * ratio);\n    this._carry = input.length - lastUsed;\n    this.port.postMessage(out);\n    return true;\n  }\n}\nregisterProcessor 'pcm16k', PCM16kProcessor);\n",
```

Comment:
Syntax + drift: `registerProcessor 'pcm16k', ...` should be `registerProcessor('pcm16k', ...)`. Also your `_carry` is a length, but you treat it like an offset; sessions will drift. Track fractional position instead (see box-filter decimator sketch in PR summary).